src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -56,6 +60,10 @@ public static void beforeClass() throws GitDownloaderException, IOException, Inv
         config = new RepoConfiguration(TEST_REPO_GIT_LOCATION, "master");
         config.setFormats(ArgsParser.DEFAULT_FORMATS);
         GitDownloader.downloadRepo(config);
+        uncommonDefaultConfig = new RepoConfiguration(TEST_REPO_UNCOMMON_DEFAULT_GIT_LOCATION,
+                CsvParser.DEFAULT_BRANCH);
+        uncommonDefaultConfig.setFormats(ArgsParser.DEFAULT_FORMATS);
+        GitDownloader.downloadRepo(uncommonDefaultConfig);
```

Comment:
This seems to only be used for 1 specific test in `CommandRunnerTest`. Doesn't really warrant a need to put this in `before()` which affects the runtime for all the other tests. Let's just put this directly for that test alone.

Reply:
```uncommonDefaultConfig``` has been moved to ```CommandRunnerTest```